ajax/libs/amalthea/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "amalthea",
+  "version": "1.1.4",
+  "author": "Apollo <zhaoyang@jizhi.im>",
+  "description": "the hyper front-end engine of data science",
+  "keywords": [
+    "html-plugin",
+    "online-judge",
+    "jupyter-notebook"
+  ],
+  "homepage": "https://amalthea.ai",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Jinglue/Amalthea.git"
+  },
+  "npmName": "amalthea",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```

Comment:
We don't host `*.gz` files which can't be used in web front-end.
This pattern will get the `*.gz` files.
Need to revise to `!(*.gz)`
Thanks.